src/System.Collections/src/System/Collections/BitArray.cs

```diff
@@ -341,6 +341,128 @@ public BitArray Not()
             return this;
         }
 
+        /*=========================================================================
+        ** Shift all the bit values to right on count bits. The current instance is
+        ** updated and returned.
+        * 
+        ** Exceptions: ArgumentOutOfRangeException if count < 0
+        =========================================================================*/
+        public BitArray RightShift(int count)
+        {
+            if (count <= 0)
+            {
+                if (count < 0)
+                {
+                    throw new ArgumentOutOfRangeException(nameof(count), count, SR.ArgumentOutOfRange_NeedNonNegNum);
+                }
+                Contract.EndContractBlock();
+
+                _version++;
+                return this;
+            }
+
+            int toIndex = 0;
+            int ints = GetArrayLength(m_length, BitsPerInt32);
+            if (count < m_length)
+            {
+                int shiftCount;
+                // int fromIndex = Math.DivRem(count, BitsPerInt32, out shiftCount); // Compilation error
```

Comment:
Here was review to use Math.DivRem but it can not be compiled. So I insert this comment.